### PR TITLE
A few fixes in the documentation

### PR DESCRIPTION
**Pull request by SuzanneSoy**

### CODE REVIEW

File: private/define-scoped-transformers.scrbl [hunk 1]
```diff
@@ -9,6 +9,6 @@ and friends.
 @defform[(define-syntax-with-scoped-pre-transformers id
            ([stx-lens pre-transformer] ...)
            transformer-expr)]{
- Binds @racket[id] as a syntax transformer equivalent to
```
Comment (jackfirth) [inline private/define-scoped-transformers.scrbl:12]: This change is a little ambiguous. The original wording was meant to state that the _syntax transformer_ was equivalent to the given expression, but separating them into two sentences makes it sound like the entire use of `define-syntax-with-scoped-pre-transformers` is equivalent to that expression.

**Comment by jackfirth** [comment]
Looks good to me pending the one prose comment, thank you!

**Comment by SuzanneSoy** [comment]
@jackfirth I edited the "equivalent to…" change. When re-reading the original version, I did have the feeling that the whole `define-syntax-with-scoped-pre-transformers` was equivalent to that expression, so I tried to make the correct meaning more obvious. Is it ok like that? Otherwise I'll revert that change tomorrow.

I also added right now parens around the `with-scoped-pre-transformers transformer-expr ([stx-lens pre-transformer] ...)`, but you may prefer the original if you're a sweet-exp user :) .

**Comment by jackfirth** [comment]
That's much better, and good catch on the parens. I like sweet expressions occasionally but would rather avoid sprinkling them in docs unnecessarily 😄

**Pull request merged**